demo/preprocessing_demo.py

```diff
@@ -0,0 +1,131 @@
+import os
+import sys
+
+import argparse
+import pandas as pd
+
+from input.admin_input import admin_input
```

Comment:
@ElettaLiride - I get a `ModuleNotFoundError: No module named 'input'` error when I run 
`python preprocessing_demo.py -h` Did you locally install WEFEDemand as a package?

Reply:
I use linux, maybe Windows is lenient with finding packages along your PYTHONPATH, however as this code will run inside a docker container, it needs to run out of the box without specific PYTHONPATH changes